SeaPublicWebsite/Services/EnergyEfficiency/PdfGeneration/PdfGenerationService.cs

```diff
@@ -58,6 +47,6 @@ public async Task<Stream> GeneratePdf(string path)
 
     private string GetLocalAddress()
     {
-        return server.Features.Get<IServerAddressesFeature>().Addresses.First();
+        return "http://localhost:80";
```

Comment:
We should leave a comment here, mentioning that port 80 is the port the app runs on in the docker container, and to remember to change it here if the docker file setting are ever changed